js/test/Exchange/test.fetchBorrowInterestHistory.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+// ----------------------------------------------------------------------------
+
+const assert = require ('assert');
+
+// ----------------------------------------------------------------------------
+
+module.exports = (exchange, code, symbol) => {
```

Comment:
this function has to be async:

- https://app.travis-ci.com/github/ccxt/ccxt/builds/244163124#L10327